File: 프로그래머스/lv1/12921. 소수 찾기/소수 찾기.java
class Solution {
    public int solution(int n) {
        int[] iArr = new int[n+1];
        int count = 0;
        for(int i = 2; i <= n; i++) {
            if(iArr[i] == 0) {
                count++;
                for(int j = i; j <= n; j = j+i) {
                    iArr[j] = 1;
                }
            }
        }
        return count;
    }
}

File: 프로그래머스/1/12921. 소수 찾기/소수 찾기.js
const solution = (n) => {
    const nums = Array(n + 1).fill(0);
    let count = 0;
    
    for(let i = 2; i <= n; i++) {
        if(nums[i] === 0) {
            count++;
            
            for(let j = i; j <= n; j = j + i) {
                nums[j] = 1;
            }
        }
    }
    
    return count;
}